PhysicsTools/Utilities/src/LumiReWeighting.cc

```diff
@@ -166,8 +166,6 @@ double LumiReWeighting::weight( const edm::EventBase &e ) {
 
     for(; PHist_iter<PHist.end() ;++PHist_iter) {
       edm::ProcessConfiguration PConf = *(PHist_iter);
-      const edm::ReleaseVersion& Release =  PConf.releaseVersion() ;
```

Comment:
@mdhildreth - please, have a look. Thanks.

Reply:
indeed, these loops appear to make no sense..

Reply:
With this change `PConf` is now an unused variable.

I checked, and non of the functions called in this loop have side-effects, so there is no useful work being done.

On the other hand, the call to `e.processHistory()` can throw if there is a problem.

Reply:
I'm updating the PR then